sdcm/cluster_aws.py

```diff
@@ -370,11 +372,14 @@ def __init__(self, ec2_ami_id, ec2_subnet_id, ec2_security_group_ids,
         self.seed_nodes_private_ips = None
         self.version = '2.1'
 
-    def add_nodes(self, count, ec2_user_data='', dc_idx=0):
+    def add_nodes(self, count, ec2_user_data='', dc_idx=0, enable_auto_bootstrap=False):
         if not ec2_user_data:
```

Comment:
When do we have this case?

Reply:
In current code, we always assign empty ec2_user_data parameter in calling add_nodes()

```
$ git grep add_nodes|grep -v def
grow_cluster_test.py:        new_nodes = self.db_cluster.add_nodes(count=add_node_cnt)
grow_cluster_test.py:            self.add_nodes(add_node_cnt)
grow_cluster_test.py:                    self.add_nodes(1)
sdcm/cluster.py:                self.add_nodes(num, dc_idx=dc_idx)
sdcm/cluster.py:            self.add_nodes(n_nodes)
sdcm/cluster.py:        raise NotImplementedError("Derived class must implement 'add_nodes' method!")
sdcm/cluster_aws.py:        added_nodes = super(ScyllaAWSCluster, self).add_nodes(count=count,
sdcm/cluster_aws.py:        added_nodes = super(ScyllaAWSCluster, self).add_nodes(count=count,
sdcm/cluster_gce.py:        added_nodes = super(ScyllaGCECluster, self).add_nodes(count=count,
sdcm/cluster_openstack.py:        added_nodes = super(ScyllaOpenStackCluster, self).add_nodes(count=count,
sdcm/nemesis.py:        new_node = self.cluster.add_nodes(count=1, dc_idx=self.target_node.dc_idx)[0]
```